### PR TITLE
carrot2 3.15.0 (new formula)

### PR DESCRIPTION
**Pull request by ankane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Hey, thanks again for all the great work on Homebrew 💯  This adds Carrot2 to the collection.

- Andrew

### CODE REVIEW

File: Formula/carrot2.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <array>
+          <string>#{opt_bin}/carrot2</string>
+        </array>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    system bin/"carrot2", "--help"
```
Comment (MikeMcQuaid) [inline Formula/carrot2.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by ankane** [comment]
Thanks for the quick response @MikeMcQuaid. I added a better test. Needed to copy everything into `testpath` due to sandbox since Carrot2 really wants to write to `libexec/logs`.

**Comment by woodruffw** [comment]
El Cap failure; looks like a race condition with the `sleep`, so perhaps make it a bit longer?

**Comment by ankane** [comment]
Thanks @woodruffw. I reread the docs and it looks like the latest version depends on Java 1.8+. When I bumped that, it fixed it for El Capitan.

**Comment by woodruffw** [comment]
Thanks! We can bump that sleep back down, then - sorry for leading you in the wrong direction.

**Comment by ankane** [comment]
Restored. No need to apologize - I didn't realize the tests were failing. I really appreciate all the work you and the other Homebrew members put it.

**Comment by woodruffw** [comment]
Thanks for contributing, @ankane!

**Pull request closed without merging**